docs/source/install/path.rst

```diff
@@ -40,21 +40,21 @@ Build from Source
 
 You choose a supported C++ compiler and configure, compile and install all missing dependencies from source.
 You are responsible to manage the right versions and configurations.
-Performance will be ideal if architecture is chosen correctly (and/or if build directly on your hardware).
+Performance will be ideal if architecture is chosen correctly (and/or if built directly on your hardware).
 You then set environment variables to find those installs.
 
 Conda
 ^^^^^
 
 We currently do not have an official conda install (yet).
-Due to pre-build binaries, performance will be sub-ideal and HPC cluster support (e.g. MPI) might be very limited.
+Due to pre-build binaries, performance could be not ideal and HPC cluster support (e.g. MPI) might be very limited.
```

Comment:
please keep `will` instead of `could`. Pre-build binaries will almost always not be tuned for the sub-architecture which will kill a lot of vectorization potential. I don't want anyone complaining about performance when using pre-build binaries, so let us be over-explicit/conservative in this teaser please.

Reply:
Ah anyway, we don't have them yet and we can complain when we do. I am fine with your version.